src/istio/mixerclient/client_impl.cc

```diff
@@ -53,14 +54,19 @@ MixerClientImpl::~MixerClientImpl() {}
 CancelFunc MixerClientImpl::Check(
     const Attributes &attributes,
     const std::vector<::istio::quota_config::Requirement> &quotas,
-    TransportCheckFunc transport, DoneFunc on_done) {
+    TransportCheckFunc transport, CheckDoneFunc on_done) {
   ++total_check_calls_;
 
   std::unique_ptr<CheckCache::CheckResult> check_result(
       new CheckCache::CheckResult);
   check_cache_->Check(attributes, check_result.get());
+
+  std::unique_ptr<CheckResponseInfo> check_response_info(new CheckResponseInfo);
```

Comment:
not need to new,  just allocate it from stack

Reply:
Done. Thanks.